### DISCUSSION TITLE
onDrop, adding a node to canvas, set the zoom to "fitview"

### DISCUSSION THREAD

**Discussion by chetan8300** [xyflow/xyflow#2969] [General]
### Describe the Bug

on dropping a node to the canvas, the canvas is set to "fitview" by default.

### Steps to Reproduce the Bug or Issue

The pro-collaborative example will reproduce the same bug by using a custom node with minimum width and height.

### Expected behavior

Expected: When I use on drop event to add a node to the canvas, it should by default only show the current zoom level of the canvas and the size of the node provided.

Issue: it sets the canvas to "fitview". Is there any workaround for that?

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by moklick** [comment]
Hey, I am not sure if I get the question right. You want to fit the view to the first added node that gets dropped on the canvas?
